Source/bmalloc/Configurations/module.modulemap

```diff
@@ -0,0 +1,55 @@
+module bmalloc {
+    requires cplusplus11
+    // List of "semi-public" bmalloc headers. What does "semi-public" mean? These are
+    // the headers which, in practice, are included by WTF headers. WTF headers are
+    // exported as part of the WebKit API, and therefore these headers need to be
```

Comment:
nit: To be more precise, I would say "exported as part of WebKit **SPI**". WTF isn't exposed to public API clients.